lib/compat/wordpress-6.1/class-gutenberg-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,224 @@
+<?php
```

Comment:
*Note to reviewers:* this file is an exact copy of core with the following changes:
- class name changed for the Gutenberg prefix.
- post_types added to the description in public function get_item_schema().
- 'postTypes'     => 'post_types', added to $key in prepare_item_for_response.

There are no other changes to the core file.